### DISCUSSION TITLE
Rozbicie na podtaski #372

### DISCUSSION THREAD

**Discussion by mattyl006** [emkarcinos/WMIAdventure#435] [Codding]
Rozbiłem na pomniejsze taski implementacje leveli na Desktop. I poszedł już pull request jednego z nich.
Jak ktoś czuje się na siłach to śmiało brać pozostałe podtaski do niego.

**Comment by itchystone** [comment]
Spoko, warto by jeszcze dodać osobny feature branch na implementacje leveli na Desktop żeby nie mergować pomniejszych tasków na maina, tylko gotowy już feature jakim są levele na desktop.

**Reply by itchystone** [reply]
Stworzyłem branch `feat_desktop_levels`

**Reply by mattyl006** [reply]
Ok
